### PR TITLE
[CODEOWNERS] Remove docs team from CODEOWNERS for unreleased changelog.

### PR DESCRIPTION
**Pull request by arthurevans**
Makes stdlib team the owners CODEOWNERS for the raw (unreleased) changelog file.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (JoeLoser) [inline .github/CODEOWNERS:7]: **Question** Since `CODEOWNERS` is effectively processed bottom to top and all of the changelog-specific changes are only for the stdlib right now (nothing else is open-source), would it instead work to add a rule

```
docs/changelog.md @modularml/mojo-standard-library
```

Comment (arthurevans) [reply .github/CODEOWNERS:7]: Oh, that would be oh so much cleaner. 🙇

**Comment by arthurevans** [comment]
Updated with @JoeLoser 's suggestion, ugliness much reduced.

**Comment by JoeLoser** [comment]
> Updated with @JoeLoser 's suggestion, ugliness much reduced.

@arthurevans thanks. Do you mind signing off your commit, please?  `git commit -s --amend` should do it.

**Comment by arthurevans** [comment]
D'oh. Signed.

**Comment by arthurevans** [comment]
As far as I understand, the failing test is a known issue unrelated to this PR.

**Review by JoeLoser — APPROVED** [review]

**Comment by JoeLoser** [comment]
LGTM.  Please update the PR description before you merge this PR.

**Pull request merged**
